src/sinks/prometheus/exporter.rs

```diff
@@ -476,12 +477,12 @@ mod tests {
         let map = &sink.metrics.read().unwrap().map;
 
         assert_eq!(
-            map.get_full(&MetricEntry(m1)).unwrap().1 .0.data.value,
```

Comment:
Wondering how that space in ` .0` ever passed the formatting checks? 😄

Reply:
It's stranger than that. When I was working with that, if I deleted the space, `rustfmt` inserted it.